nominees/berkely-air-sums.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "Berkely Air Sums",
+    "description": "The Stove Use Monitoring System (SUMS) provides insights into cookstove usage patterns, number of meals cooked, and time of use by recording stove temperature changes. Data sampled by SUMS are uploaded to a computer via a data cable and further analyzed using proprietary algorithms developed to quantify cookstove usage in households.",
+    "website": "https://berkeleyair.com/",
+    "license": [
+      {
+        "spdx": "",
+        "licenseURL": ""
+      }
+    ],
+    "SDGs": [
+      {
+        "SDGNumber": 3,
+        "evidenceURL": "http://berkeleyair.com/about-us/mission/"
+      },
+      {
+        "SDGNumber": 7,
+        "evidenceURL": "https://berkeleyair.com/monitoring-instruments-sales-rentals/stove-use-monitoring-system-sums/"
+      },
+      {
+        "SDGNumber": 9,
+        "evidenceURL": "https://berkeleyair.com/monitoring-instruments-sales-rentals/stove-use-monitoring-system-sums/"
+      }
+    ],
+    "sectors": [],
+    "type": [
+      "software"
+    ],
+    "organizations": [
+      {
+        "name": "Berkely Air Monitoring Group",
```

Comment:
@ijeomaemeruwa Kindly correct the spelling of the name field to `Berkeley Air Monitoring Group`